code/game/objects/structures/dropship_ammo.dm

```diff
@@ -287,14 +282,12 @@
 	ammo_id = "n"
 	point_cost = 500
 
-	detonate_on(turf/impact)
-		impact.ceiling_debris_check(3)
-		spawn(5)
-			explosion(impact,1,2,3,6,1,0) //relatively weak
-			for(var/turf/T in range(4,impact))
-				T.ignite(60, 30) //cooking for a long time
-			qdel(src)
-
+/obj/structure/ship_ammo/rocket/napalm/detonate_on(turf/impact)
+	impact.ceiling_debris_check(3)
+	explosion(impact, 2, 3, 4, 6) //relatively weak
+	for(var/turf/T in range(4, impact))
+		T.ignite(60, 30) //cooking for a long time
```

Comment:
could this use flame_turf?

Reply:
That is a gun proc, though, no?
Fire does need a bit of a refactor...

Reply:
Used `flame_radius` which does what you requested, I believe.

Reply:
I meant so that you could remove the turf loop here.

Reply:
oops, I meant to do that